src/com/company/Board.java

```diff
@@ -0,0 +1,45 @@
+package com.company;
+
+public class Board {
+    private final int size;
+    private final PlayerSignType[][] data;
+
+    public Board(final int size) {
+        this.size = size;
+        data = new PlayerSignType[size][size];
+        clear();
+    }
+
+    private void clear() {
+        for (int i = 0; i < data.length; i++) {
+            for (int j = 0; j < data[i].length; j++) {
+                data[i][j] = PlayerSignType.EMPTY;
+            }
+        }
+    }
+
+    public boolean insertSign(final PlayerSignType sign, final int row, final int column) {
+        if (row > 0 && row < size
+                && column > 0 && column < size
+                && data[row][column] == PlayerSignType.EMPTY) {
+            data[row][column] = sign;
+            return true;
+        }
+        return false;
+    }
+
+    public String getSignAndChangeToString(final int row, final int column) {
+        if (data[row][column] == PlayerSignType.EMPTY) {
+            return (" ");
+        } else {
+            return data[row][column].toString();
+        }
+    }
+
+    public boolean isSignEqual(final PlayerSignType sign, final int row, final int column) {
```

Comment:
Ta metoda mogłaby mieć jedną linjkę. Popraw proszę. 
Dodatkjowo dodam, że enumów nie trzeba porównywać przez equals bo każdy jest dokładnie jeden w całym programie. (zresztą w metodzie powyżej używasz właśnie `==` w takiem samym przypadku)

Reply:
return data[row][column].toString();
ale wtedy zamiast spacji mam napis "empty", i nie mogę sobie zrobić tego co pochwaliłeś czyli
lineWithData.append(" ").append(board.getSignAndChangeToString(i, j)).append(" |");
bo w tym miejscu muszę mieć jeden znak.
Dopisałam osobną funkcję do zmiany znaku na spację. Ale dopisałam 7 linijek a skasowałam (zakomentowałam) 5, więc nie wiem czy tu jest oszczędność;)

Reply:
Chyba mówisz o metodzie wyżej. Patrz dobrze której linijki dotyczy komentarz. To było do `isSignEqual` a nie do `getSignAndChangeToString`